codegen/templates/http_client.tmpl

```diff
@@ -238,8 +238,11 @@ func (c *{{$clientName}}) {{$methodName}}(
 	if (c.circuitBreakerDisabled) {
 		res, err = req.Do()
 	} else {
-		err = hystrix.DoC(ctx, "{{$clientID}}", func(ctx context.Context) error {
+		hystrix.DoC(ctx, "{{$clientID}}", func(ctx context.Context) error {
```

Comment:
I think you want to keep the err arg here and change the var name on line# 242 below.